kustomize/kustomization.yaml

```diff
@@ -22,3 +22,4 @@ components:
   # - components/cymbal-branding
   # - components/google-cloud-operations
   # - components/memorystore
+  # - components/network-policy
```

Comment:
The goal is to a add each and every Kustomize Component here:
```
  # - components/name-of-component
```
for convenience.